MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/MvvmCross.Droid.Support.V7.RecyclerView.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
oops, private package nuspec landed in repo. Sorry! will remove that in a while.